ProductApp/src/main/java/com/stack/jspdemo/repository/ProductRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.stack.jspdemo.repository;
+
+import org.springframework.data.repository.CrudRepository;
+
+import com.stack.jspdemo.model.Product;
+
+public interface ProductRepository extends CrudRepository<Product, Long>, ProductRepositoryCustom{
```

Comment:
Why not implementing instead `JpaRepository<E, R>` ? 
This interface already provides us a `findAll()` method, so it wouldn't be necessary to implement a `getAllProducts()` method